release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1381,6 +1395,15 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Storage*
 
+* Previously, the recycler-pod template was incorrectly placed in the kubelet static manifest directory. This incorrect location produced kubelet static pod log messages indicating a recycler static pod start failure. With this update, the misplaced recycler-pod template has been removed from the kubelet static pod manifest directory. As a result, the error messages no longer appear. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896226[*BZ#1896226*])
```

Comment:
s/ kubelet static pod/static pod (I don't see _kubelet static pod_ and here there are a lot of words qualifying _messages_) 
s/indicating/that indicated